### ISSUE TITLE
Reverse transform of column names when inserting or updating

### ISSUE THREAD

**Issue by davojan** [porsager/postgres#16]
Transforming column names in both ways would greatly improved ergonomics of the library. Consider this example:
```js
const sql = postgres({
  transform: {
    column: postgres.toCamel,
  },
}) 
const entity = {
  appliedAt: new Date,  // db column name is "applied_at"
}
// error: column "appliedat" of relation "some_table" does not exist
sql`INSERT INTO some_table ${sql(entity)}`
```
Transform option should work both ways, I think. May be configured like this:
```js
{
  transform: {
    column: {
      fromDb: postgres.toCamel,
      toDb: postgres.toSnake,
    },
  },
}
```

**Comment by porsager** [comment]
Hi @davojan ..

Yes I have considered an option for that.. Good to track in this issue - thanks ;)

**Comment by porsager** [comment]
I like the non breaking way you suggested of supplying an object with `{ to, from }`

**Comment by xiaoyu-tamu** [comment]
It would be useful!

**Comment by xiaoyu-tamu** [comment]
Should we also transform column name for `sql(obj, ...columns_names)` ?

user can use `sql({firstName:'peter'}, 'firstName')`  instead of `sql({firstName:'peter'}, 'first_name')`.

**Comment by iby** [comment]
@porsager Hey! Need this one badly. Has anyone worked on this? Are you open towards the PR? P.S. Also love the idea with `{ to, from }`.

**Comment by basaran** [comment]
this could be extremely useful.

**Comment by porsager** [comment]
@iby and @basaran Would you mind trying out how https://github.com/porsager/postgres/pull/212 works for you?

For eg. camel casing use:
```js
const sql = postgres({
  transform: {
    column: { to: postgres.fromCamel, from: postgres.toCamel }
  }
})
```

**Comment by basaran** [comment]
Hello,

Thank you, I think it's fine:

```sql
CREATE TABLE IF NOT EXISTS public."camelTable"
(
    snake_to_camel text COLLATE pg_catalog."default"
)
```

```js
async function dooo() {
    const importantInformation = {
        snakeToCamel: "aaaa",
    };

    const importantColumns = ["snakeToCamel"];

    const a_insert = await sql`
      INSERT INTO "camelTable" ${sql(importantInformation)}
    `;

    const b_select = await sql`
      SELECT ${sql(importantColumns)} FROM "camelTable" 
    `;

    console.log(a_insert);
    console.log(b_select);
}
```

```sql
INSERT INTO "camelTable" ("snake_to_camel") values ($1)
[{"type":0,"value":"aaaa","raw":"aaaa"}]
SELECT "snake_to_camel" FROM "camelTable"
[
  count: 1,
  command: 'INSERT',
  state: { status: 'I', pid: 1634864, secret: 1829386482 }
]
[
  { snakeToCamel: 'aaaa' },
  { snakeToCamel: 'aaaa' },
  columns: [ { name: 'snakeToCamel', parser: undefined, type: 25 } ],
]
```

**Comment by porsager** [comment]
Fixed with #212
